discovery-service/src/main/java/org/zowe/apiml/discovery/ApimlInstanceRegistry.java

```diff
@@ -143,6 +146,10 @@ protected int resolveInstanceLeaseDurationRewritten(final InstanceInfo info) {
 
     @Override
     public void register(InstanceInfo info, int leaseDuration, boolean isReplication) {
```

Comment:
what happens when service tries to unregister by original id?

Reply:
I've tried to register the sample service which service id then modified, afterwards shutdown it and it successfully unregistered.